src/config.ts

```diff
@@ -30,6 +30,7 @@ const validators = {
   REACT_APP_GEM_API_KEY: str(),
   REACT_APP_FRIENDLY_CAPTCHA_SITE_KEY: str(),
   REACT_APP_FOXY_APY: num({ default: 0.15 }),
+  REACT_APP_ETH_FOX_APR: num({ default: 60 }),
```

Comment:
please change this to be zero based, like FOXY_APY is for consistency. the math is easier, gives us more precision, is consistent etc